### DISCUSSION TITLE
deferrable operator no triggering

### DISCUSSION THREAD

**Discussion by flo-th** [apache/airflow#29361] [Q&A]
Hi all,

Trying to implement a simple deferrable operator based on this [example](https://airflow.apache.org/docs/apache-airflow/stable/authoring-and-scheduling/deferring.html), nothing seems to appear after the manual triggering of my DAG (same case with the exact code of example).

```
class TestDefer(BaseOperator):
    def execute(self, context):
        print("--- execute --")

        self.defer(
            trigger=TimeDeltaTrigger(delta=timedelta(seconds=1)),
            method_name="func",
        )

    def func(self, context, event=None):
        print("--- func ----")
        pass

with DAG(
    "def_dag", schedule_interval=None, start_date=datetime.now(),
) as dag:
    t = TestDefer(task_id="defer_task")
```

and then :  
    _airflow dags test  def_dag now_
    _airflow triggerer_

Result : _func_ is never called.

Thanks in advance for your help.

**Comment by potiuk** [comment]
The dags test command does not work with deferrable operators. You have to run it as regular DAG.
